dev/io.openliberty.persistence.3.0.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/ValidationExceptionResource.nlsprops

```diff
@@ -631,3 +631,5 @@
 
 7356=Procedure: [{1}] cannot be executed because {0} does not currently support multiple out parameters
 
+7357=URL [{0}] for resource [{1}] does not belong to a valid persistence root (as per JPA Specification)
```

Comment:
Change to `The [{0}] URL for the [{1}]  resource does not belong to a valid persistence root (as in the JPA Specification).`

Reply:
What about "The URL: "[{0}]", for resource: "[{1}]", does not ..."?

Reply:
The preferred syntax is "the _variable_ _noun_" not "_noun_ _variable_". Please do not add a colon.